modules/swagger-codegen/src/main/resources/python/swagger.mustache

```diff
@@ -107,34 +107,34 @@ class ApiClient:
     else:
       return urllib.quote(str(obj))
 
-  def sanitizeForSerialization(self, obj):
-    """Dump an object into JSON for POSTing."""
-
-    if type(obj) == type(None):
+  @staticmethod
+  def sanitizeForSerialization(obj):
+    """
+    Sanitize an object for Request.
+
+    If obj is None, return None.
+    If obj is str, int, long, float, bool, return directly.
+    If obj is datetime.datetime, datetime.date convert to string in iso8601 format.
+    If obj is list, santize each element in the list.
+    If obj is dict, return the dict.
+    If obj is swagger model, return the properties dict.
+    """
+    if isinstance(obj, type(None)):
       return None
-    elif type(obj) in [str, int, long, float, bool]:
+    elif isinstance(obj, (str, int, long, float, bool, file)):
```

Comment:
`isinstance` is better than `type`
ref: http://docs.quantifiedcode.com/python-code-patterns/readability/do_not_compare_types_use_isinstance.html?highlight=isinstance